packages/api/src/hooks/useCreateOtherCFDAccount.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+import useRequest from '../useRequest';
+import useInvalidateQuery from '../useInvalidateQuery';
+
+/** A custom hook that creates the Other CFD account. */
+const useCreateOtherCFDAccount = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useRequest('trading_platform_new_account', {
+        onSuccess: () => {
+            invalidate('trading_platform_accounts');
+        },
+    });
+
+    // Add additional information to the create Other CFD account response.
+    const modified_data = useMemo(() => {
+        if (!data) return undefined;
+
+        return { ...data };
+    }, [data]);
+
+    return {
+        /** The response and the mutation of the create Other CFD account API request */
+        data: modified_data,
+        ...rest,
+    };
+};
+
+export default useCreateOtherCFDAccount;
```

Comment:
@hamza-deriv The name is a bit weird, Like what is `other`, But I don't have any suggestions 😬

Reply:
Its because in Compare Account and in Traders Hub we refer as `MT5` or `Other CFD Platforms`. Thats why i use the name `otherCFD` 😆

Reply:
![image](https://github.com/binary-com/deriv-app/assets/120543468/9abf1ad6-c945-46cf-bc9f-8727af7c546a)

Reply:
Maybe something like `useCreateThidPartyCFDAccount` or something like that, I don't know 😅 Let's move on, Later we can simply rename it once we have a better name for it 🙇🏻 

Maybe @thisyahlen-deriv can help to come up with a more meaningful name.

Reply:
![Screenshot 2023-09-05 at 2 50 16 PM](https://github.com/binary-com/deriv-app/assets/104053934/ac79a53f-f4e8-4c48-ab08-e922f19142aa)

Reply:
`useFreshCFDAccount` sounds fine to me